sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -76,7 +76,11 @@ describe("EventHub Receiver", function(): void {
   });
 
   describe("with EventPosition specified as", function(): void {
-    it("'from end of stream' should receive messages correctly ", async function(): Promise<void> {
+    
+    // TODO: Enable following test as part of https://github.com/Azure/azure-sdk-for-js/issues/3714
+    // After the implementation of BatchingReceiver has been fixed
```

Comment:
`After the implementation of BatchingReceiver has been fixed`
->
`After we ensure the multiple receiveBatch calls on the user facing receiver work as expected`